packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -15,30 +60,162 @@ export default util.createRule({
     messages: {
       missingAccessibility:
         'Missing accessibility modifier on {{type}} {{name}}.',
+      unwantedPublicAccessibility:
+        'Public accessibility modifier on {{type}} {{name}}.',
     },
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          noPublic: {
+            type: 'boolean',
+          },
+          overrides: {
+            type: 'object',
+            properties: {
+              accessors: override,
+              constructors: override,
+              methods: override,
+              properties: override,
+              parameterProperties: override,
+            },
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
   },
-  defaultOptions: [],
+  defaultOptions: [
+    // technically there is a default, but the overrides mean
+    // that if we apply them here, it will break the no override case.
+    {},
+  ],
   create(context) {
+    const option: Config = util.applyDefault([{}], context.options)[0];
```

Comment:
you can use the second argument of the create instead:
```
create(context, [option]) {
```

Reply:
Neat! Changed :)